lib/twitter_cldr/js/compiler.rb

```diff
@@ -59,14 +60,22 @@ def compile_bundle(bundle, bundle_elements, bundle_hash, options = {})
 
       def compile_each(options = {})
         @locales.each do |locale|
-          bundle = TwitterCldr::Js::Renderers::Bundle.new
+          bundle = TwitterCldr::Js::Renderers::DataBundle.new
           bundle[:locale] = locale
-          file = compile_bundle(bundle, @features, implementation_renderers, options)
+          file = compile_bundle(bundle, @data, data_renderers, options)
 
           yield file, TwitterCldr.twitter_locale(locale)
         end
       end
 
+      def compile_implementation(options = {})
+        bundle = TwitterCldr::Js::Renderers::Bundle.new
+        bundle[:locale] = :en # setting a default locale.
```

Comment:
I doubt this will ever change, but for the sake of having defaults in one place, you can use `TwitterCldr::DEFAULT_LOCALE` here.